lms/services/canvas_api/_basic.py

```diff
@@ -50,18 +50,34 @@ def send(
         params=None,
         headers=None,
         url_stub="/api/v1",
+        timeout=(10, 10),
     ):
         """
         Make a request to the Canvas API and apply a schema to the response.
 
         :param method: HTTP method to use
+
         :param path: Path fragment to add to the end of the URL
+
         :param schema: A schema object which this request must
+
         :param params: URL parameters to include
+
         :param headers: Headers to include
+
         :param url_stub: Path prefix
-        :return: The result of applying the schema to the response
+
+        :param timeout: The timeout to pass to requests
+        :type timeout: Either a (connect_timeout, read_timeout) 2-tuple or a
+            single number that will be used for both the connect timeout and
+            the read timeout. The values are in seconds and can be either ints
+            or floats. See:
+            https://docs.python-requests.org/en/latest/user/quickstart/#timeouts
+            https://docs.python-requests.org/en/latest/user/advanced/#timeouts
```

Comment:
:man_shrugging: I think the other option would be to remove the whole docstring from `AuthenticatedClient` and have it just refer the reader to `BasicClient`'s docstring. (That's what `OAuthHTTPService` does with `HTTPService`.)